build/Glyssen.proj

```diff
@@ -117,7 +117,7 @@
 		which itself contains tests. Let's move the test scaffolding in SIL.DblBundle.Tests
 		into a separate test scaffolding dll
 		Note: the following line needs to use backslashes; otherwise, the Exclude directive doesn't work. -->
-		<CreateItem Include="$(RootDir)\output\$(Configuration)\*Tests.dll" Exclude="$(RootDir)\output\$(Configuration)\SIL.DblBundle.Tests.dll">
+		<CreateItem Include="$(RootDir)\output\$(Configuration)\**\*Tests.dll" Exclude="$(RootDir)\output\$(Configuration)\SIL.DblBundle.Tests.dll">
```

Comment:
Was this change actually needed? Seems the GlyssenEngineTests follow the same pattern as the others.